modules/core/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -400,4 +400,15 @@ public static class Compute {
         /** Class loader error. */
         public static final int CLASS_LOADER_ERR = COMPUTE_ERR_GROUP.registerErrorCode((short) 2);
     }
+
+    /**
+     * File transfer error group.
+     */
+    public static class FileTransfer {
```

Comment:
Do we really need a dedicated error group for this? Perhaps, it should be a part of the network error group. WDYT?